iuriMorari/src/main/java/AbstractUser.java

```diff
@@ -0,0 +1,48 @@
+import java.util.ArrayList;
```

Comment:
btw, these classes are not from previous your previous PR, please check and revert